backend/src/main/java/com/ody/meeting/controller/MeetingController.java

```diff
@@ -83,9 +89,11 @@ public ResponseEntity<MeetingSaveResponse> save(
             @AuthMember Member member,
             @RequestBody MeetingSaveRequest meetingSaveRequest
```

Comment:
[요청] save 메서드 매개변수에 `@Valid`를 추가해주세요 아마 머지 과정에서 누락한 것 같아요 😓

```suggestion
            @Valid @RequestBody MeetingSaveRequest meetingSaveRequest
```

Reply:
<img width="1059" alt="image" src="https://github.com/user-attachments/assets/f23d24d2-32a2-469f-bbb2-89556fed2ff2">
이미 커밋한 제 코드에는 반영이 되어 있고
[모임 개설 커밋](https://github.com/woowacourse-teams/2024-ody/pull/121/commits/68e64715ba6e0f431411b025511c069a9b55e260)
해당 커밋에도 반영이 되어 있는데 왜 저렇게 코드가 나오는지 모르겠네요 😥

변경사항인 없는 부분이라 커밋도 안되서 어떤 문제로 머지 전 코드가 반영되게 된건지 알아봐야할 듯하네요 ..